lab_4_retrieval_w_clustering/main.py

```diff
@@ -81,6 +104,16 @@ def _calculate_bm25(self, tokenized_document: list[str]) -> Vector:
         Returns:
             Vector: BM25 vector for document.
         """
+        if not tokenized_document or not isinstance(tokenized_document,list) or not\
+                all(isinstance(paragraph,str) for paragraph in tokenized_document):
+            raise ValueError
+        vector_list = [0.0 * i for i in range(len(self._vocabulary))]
+        bm25 = calculate_bm25(self._vocabulary, tokenized_document, self._idf_values, 1.5, 0.75, self._avg_doc_len,
+                                 len(tokenized_document))
```

Comment:
do you really need to pass here default arguments?

Reply:
I don't really, but doesn't it look prettier with them in here?

Reply:
their values ​​are indicated precisely so that when called they are set by default, so rm

Reply:
removed